test/JDBC/expected/table-variable-vu-verify.out

```diff
@@ -204,3 +204,12 @@ int
 1
 ~~END~~
 
+
+-- double-check that the underlying type for table_variable_vu_type is pass-by-val
+select typbyval from pg_type where typname = 'table_variable_vu_type';
```

Comment:
For pg_type field checks, please also add the following test cases:
1. `create type custom_schema.table_type as table ...`
2. `declare @tab_var as table ...`
3. table types created by TVF

Reply:
Table types from TVF are covered in TestTableType MVU tests:
```
create function TestTableType_vu_prepare_func1 (@number int, @tv TestTableType_vu_prepare_t4 READONLY) returns table as return (
select *, @number from @tv 
);
go
```
I will add tests for the other two cases

Reply:
TestTableType test does not check typbyval field. Please check typbyval field directly, e.g.,
```
select typbyval from pg_type where typname like '@testtabletype_vu_prepare_func1%';
go

declare @tableVar as table (x int)
insert into @tableVar values (1),(2),(3)
select * from @tableVar;
select typbyval from pg_catalog.pg_type where typname like '@tablevar%';
go
```

Reply:
Added a check for @tableVar. However, it appears that TVFs do not generate a pg_type entry, as their return type is set to `record` type, which is a built-in PG type.

Reply:
Added example with TVF